robot/EDA/tests/browser/settings/system.robot

```diff
@@ -33,6 +54,48 @@ Verify system settings can retain value on save
     ...                       Automatically Rename Household Accounts=true
     Verify dropdown value
     ...                       Default Account Model=Household Account
+    Verify system dropdown value
+    ...     The name format used for auto-created Administrative Accounts.={!LastName} Admin Account
+    ...     The format used for naming Household Accounts.={!LastName} House Account
+    Click action button on EDA settings page    Edit
+    Update system dropdown value
+    ...                       Administrative Account Name Format={!LastName} Administrative Account
+    ...                       Household Account Name Format={!{!FirstName}} {!LastName} Family
+    Click action button on EDA settings page    Save
+    Go to EDA settings tab    System
+    Verify dropdown value
+    ...                       Administrative Account Name Format={!LastName} Administrative Account
+    ...                       Household Account Name Format={!{!FirstName}} {!LastName} Family
+
+Verify Error Notification Recipients field picklist availability
+    [Documentation]         Updates 'Error Notification Recipients' to chatter group and user and
+    ...                     validates if the SFID values of the same are retained upon saving.
+    [tags]                    unstable        W-8134943     rbt:high
+    Go to EDA settings tab    System
+    Click action button on EDA settings page    Edit
+    Select recipient type value
+    ...                       Select recipient type:=Chatter Group
+    Log     ${group_name}   console=true
```

Comment:
If you want to commit this to MetaCI, I'd suggest you remove the console arg. I suggested that more for debugging on ur local. But on MetaCI, we prefer not to log to console.

Reply:
Forgot to remove the debug statements and its not needed. I have removed it